lib/iris/tests/unit/fileformats/grib/message/test_GribMessage.py

```diff
@@ -101,7 +99,7 @@ def test_bitmap_present(self):
                                       6: {'bitMapIndicator': 0,
                                           'bitmap': self.bitmap},
                                       7: {'codedValues': input_values}})
-        result = message.data.masked_array()
+        result = as_concrete_data(message.data)
         expected = np.ma.masked_array(output_values,
                                       np.logical_not(self.bitmap))
         expected = expected.reshape(self.shape)
```

Comment:
This is now fixed and the tests are passing. But I'm not convinced that we have enough test coverage of testing how missing data is handled, considering the fact when missing data wasn't being handled correctly only one (1) test failed.